docs/source/preferences/PreferencesInEnvisage.rst

```diff
@@ -1,3 +1,5 @@
+.. _preferences-in-envisage:
+
```

Comment:
Orthogonal... this section looks like it belongs to Envisage? Maybe a separate issue here.